src/BackgroundDimmer/BackgroundDimmer.tsx

```diff
@@ -35,14 +19,26 @@ export type BackgroundDimmerProps = {
 };
 
 export type BackgroundDimmerDefaultProps = {
+  /** background-color */
   color: string,
+  /** opacity */
   alpha: number,
+  /** z-index (BackgroundDimmer has `position: fixed`) */
   zIndex: number,
+  /** centeredContentWrapper width */
   width: string | number,
+  /** centeredContentWrapper max-width */
   maxWidth: string | number,
+  /** centeredContentWrapper height */
   height: string | number,
-  maxHeight: string | number
-}
+  /** centeredContentWrapper max-height */
+  maxHeight: string | number,
+  /** avoid propagation for scroll events */
+  stopScrollPropagation: boolean,
```

Comment:
This was implicitly `undefined`, I explicitly made it `false` as a default prop.